web/src/mocks/handlers.tsx

```diff
@@ -2,13 +2,6 @@
 import {rest} from "msw";
 
 export const handlers = [
-    rest.post('*', (req, res, ctx) => {
```

Comment:
This was catch all posts before, MSW will throw errors when a request is made that isn't intercepted anyway, so this didn't give us anything